client/gribiclient.go

```diff
@@ -345,7 +347,23 @@ func (c *Client) Connect(ctx context.Context) error {
 				log.V(2).Infof("shutting down recv goroutine")
 				return
 			}
-			if done := respHandler(stream.Recv()); done {
+			resp, err := stream.Recv()
+			trailer := stream.Trailer()
+
+			if len(trailer) != 0 {
+				if url, ok := trailer["errordetails-typeurl"]; ok {
```

Comment:
This is a magic string - right? Can we make it a constant outside of this code please? I suggested the name `GRIBIErrorTrailingMetadataName` above.

Reply:
Defined the magic strings as constants